tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -238,6 +241,79 @@ def test_square():
 
     npt.assert_almost_equal(result, numpy_op)
 
+
+def _assert_sym_equal(lhs, rhs):
+    assert lhs.list_inputs() == rhs.list_inputs()  # input names must be identical
+    assert len(lhs.list_outputs()) == len(rhs.list_outputs())  # number of outputs must be identical
```

Comment:
should we assert for more things here,  like output_shape?

Reply:
I thought that's unnecessary given the test in _check_onnx_export() which sends random data into the original network and into imported network and makes sure outputs are equal. That includes shape and values.